install-terraform/action.yml

```diff
@@ -0,0 +1,25 @@
+name: "Terraform Setup"
+description: "Installs Terraform"
+inputs:
+  tf_version:
+    description: "Terraform version to install"
+runs:
+  using: "composite"
+  steps:
+      - name: "Install Terraform ${{ inputs.tf_version }}"
+        shell: bash
+        run: |
+            wget \
+            -q \
+            -O ~/bin/terraformzip.zip \
+            "https://releases.hashicorp.com/terraform/${{ inputs.tf_version }}/terraform_${{ inputs.tf_version }}_linux_amd64.zip"
+            unzip -o ~/bin/terraformzip.zip -d /tmp
```

Comment:
If you're unzipping to `/tmp`, don't you need a `mv /tmp/terraform-bla/terraform ~/bin` somewhere?